.travis.yml

```diff
@@ -63,6 +63,16 @@ matrix:
         # Make sure that the README is formatted properly.
         - cd python
         - python setup.py check --restructuredtext --strict --metadata
+        - cd ..
+        # Run Bazel linter Buildifier.
+        - wget https://dl.google.com/go/go1.10.3.linux-amd64.tar.gz
+        - tar -xf go1.10.3.linux-amd64.tar.gz
+        - mkdir $HOME/go_dir
+        - export GOROOT=`pwd`/go
+        - export GOPATH="$HOME/go_dir"
+        - export PATH="$GOROOT/bin:$GOPATH/bin:$PATH"
```

Comment:
I don't think we need to manually install golang. Travis should have pre-installed golang env.  
Here's an example https://gist.github.com/y0ssar1an/df2dab474520c4086926f672c52db139.

Reply:
I found pre-installed golang version is 1.7.3, which is not compatible with buildifier.